src/pages/start/6.init-flake.mdx

```diff
@@ -17,7 +17,7 @@ Turning your own projects into flakes can be somewhat tricky, so we at [Determin
 You can run `fh init` using Nix:
 
 ```shell
-nix run "https://flakehub.com/f/NixOS/nixpkgs/*.tar.gz#fh" -- init
+nix run "https://flakehub.com/f/DeterminateSystems/fh/*" -- init
```

Comment:
```suggestion
nix run "https://flakehub.com/f/DeterminateSystems/fh/0.1" -- init
```

How about using `0.1`?

Reply:
I'm not so keen on that because there's no reason in principle we couldn't release a 0.2 and we may well forget to update this in that case